include/cuco/detail/open_addressing/open_addressing_impl.cuh

```diff
@@ -834,7 +835,7 @@ class open_addressing_impl {
     using temp_allocator_type =
       typename std::allocator_traits<allocator_type>::template rebind_alloc<char>;
 
-    cuco::detail::index_type constexpr stride = std::numeric_limits<int32_t>::max();
+    cuco::detail::index_type constexpr stride = std::numeric_limits<std::int32_t>::max();
```

Comment:
Stride is also used on the GPU, right? Should it be `cuda::std::` instead, then?